docs/core/tools/global-tools.md

```diff
@@ -120,39 +120,66 @@ dotnet --list-runtimes
 
 Contact the author of the Global Tool and see if they can recompile and republish their tool package to NuGet with an updated version number. Once they have updated the package on NuGet, you can update your copy.
 
-The .NET Core CLI tries to add the default locations to the PATH environment variable on its first usage. However, there are a couple of scenarios where the location might not be added to PATH automatically, such as:
+The .NET Core CLI tries to add the default locations to the `PATH` environment variable on its first usage. `PATH` isn't updated on macOS if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
 
-* If you've set the `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` environment variable.
+<!-- 
+However, there are a couple of scenarios where the location might not be added to PATH automatically, such as:
+
+* If you've set the `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` environment variable. 
 * On macOS, if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
 * On Linux, you need to edit the shell environment file to configure the PATH.
+Linux is no longer a problem when installed on bash with .deb and .rpm - see https://dotnet.microsoft.com/download/linux-package-manager/ubuntu16-04/sdk-current
+-->
+
+## Tools in the .NET Core SDK
+
+The .NET Core SDK contains the following tools:
 
-## Other CLI commands
+* [dev-certs](/aspnet/core/security/enforcing-ssl#trust) : Must be installed.
+* [ef](/ef/core/miscellaneous/cli/dotnet)
+* [sql-cache](/aspnet/core/performance/caching/distributed#distributed-sql-server-cache)
+* [user-secrets](/aspnet/core/security/app-secrets)
+* [dotnet-watch](/aspnet/core/tutorials/dotnet-watch)
 
-The .NET Core SDK contains other commands that support .NET Core Global Tools. Use any of the `dotnet tool` commands with one of the following options:
+[dotnet-aspnet-codegenerator](dotnet-aspnet-codegenerator.md) is global tool that can be installed. For example, the following command installs the ASP.NET Core scaffolding tool:
+
+```console
+dotnet tool install --global dotnet-aspnet-codegenerator
+```
+
+`dotnet tool` commands support the following options:
 
 * `--global` or `-g` specifies that the command is applicable to user-wide Global Tools.
 * `--tool-path` specifies a custom location for Global Tools.
 
-To find out which commands are available for Global Tools:
+The following command shows which commands are available for Global Tools:
```

Comment:
This is not correct! You cannot determine what Global Tools are available to install via NuGet. We are working on that but it will not be via --help.